imapclient/imapclient.py

```diff
@@ -472,6 +476,22 @@ def select_folder(self, folder, readonly=False):
         self._command_and_check('select', self._normalise_folder(folder), readonly)
         return self._process_select_response(self._imap.untagged_responses)
 
+    def unselect_folder(self):
+        """Unselect the current folder and release ressources associated.
+
+        Unlike ``close_folder``, the ``UNSELECT`` command does not expunge
+        the mailbox, keeping messages with \Deleted flags for example.
+
+        The return value is the server command response message
+        followed by a list of status responses, which should be empty.
```

Comment:
When wouldn't the list of status responses be empty? Why return it if it's always empty?